sections/classification.md

```diff
@@ -166,7 +166,7 @@ df['label'].value_counts()
 
 
 ## What if we want to visualize that information?
-We first create a `figure` and `axes` on which to draw our charts using `plt.subplots()`. Each chart is one axes, and a figure can contain multiple charts. Our data is encapsulated in `df['label'].value_counts()`, which is itself a dataframe. We then tell the Pandas to visualize the dataframe as a bar chart using `.plot.bar(ax=ax, rot=0)`. The `ax` keyword tells Pandas which chart in the figure to plot, and the `rot` keyword controls the rotation of the x axis labels.
```

Comment:
Whoops! Actually keep the first "axes" as is, plural, and the second "axes" change to "axis", singular (although I may be misunderstanding the meaning here)